From fba082dd095ff98a3b00b6173df9a8d3bd5ce192 Mon Sep 17 00:00:00 2001
From: SF Markus Elfring <elfring@users.sourceforge.net>
Date: Mon, 5 Mar 2018 16:21:26 -0800
Subject: [PATCH 011/205] memory-EMIF: Use seq_putc() in emif_regdump_show()

This commit comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

A single character (line break) should be put into a sequence.
Thus use the corresponding function "seq_putc".

This issue was detected by using the Coccinelle software.

Signed-off-by: Markus Elfring <elfring@users.sourceforge.net>
Signed-off-by: Santosh Shilimkar <santosh.shilimkar@oracle.com>
(cherry picked from commit d363a88b31fc03268eada426f5940ef685b8df21)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/memory/emif.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/memory/emif.c b/drivers/memory/emif.c
index 04644e7..2f21444 100644
--- a/drivers/memory/emif.c
+++ b/drivers/memory/emif.c
@@ -127,7 +127,7 @@ static int emif_regdump_show(struct seq_file *s, void *unused)
 
 	for (i = 0; i < EMIF_MAX_NUM_FREQUENCIES && regs_cache[i]; i++) {
 		do_emif_regdump_show(s, emif, regs_cache[i]);
-		seq_printf(s, "\n");
+		seq_putc(s, '\n');
 	}
 
 	return 0;
-- 
1.7.9.5

